libs/cart/src/selectors/cart/cart.selector.ts

```diff
@@ -20,9 +20,132 @@ export interface DaffCartStateMemoizedSelectors<
 > {
 	selectCartState: MemoizedSelector<object, DaffCartReducerState<T>>;
 	selectCartValue: MemoizedSelector<object, T>;
-  selectCartLoading: MemoizedSelector<object, boolean>;
   selectCartResolved: MemoizedSelector<object, boolean>;
 
+  /**
+   * The object that holds all the loading states for cart operations.
+   */
+  selectCartLoadingObject: MemoizedSelector<object, DaffCartReducerState<T>['loading']>;
+  /**
+   * Selects whether there is a cart operation in progress.
+   * This does not include operations specifally for cart subfields.
```

Comment:
Maybe this should cover all operations for cart? Like it returns true if any subfields of cart are Resolving or Mutating. I can't think of a use case where the entire cart is loading, and we wouldn't want a subfield of the cart to be loading.

Reply:
Either that or we add another selector that tracks if any of the feature areas are loading. I kinda like having a disctinction between "are any of the cart fields loading" and "are all of the cart fields loading".

Reply:
That sounds fine to me, as long as the names makes sense

Reply:
How about I use `selectCartFeatureLoading` to refer to the "are any of the cart fields loading"?

Reply:
sure, sounds good

Reply:
added